shadow-cljs.edn

```diff
@@ -95,7 +95,8 @@
                             ;; otherwise sometimes you'll get weird errors when
                             ;; instrumenting functions.
                             :static-fns         false
-                            :infer-externs      true}
+                            :infer-externs      true
+                            :reader-features    #{:mobile}}
```

Comment:
Here we're using a shadow-cljs feature for conditionally requiring the metrics interceptor logic for all the builds except the component-tests. This is because requiring the metrics interceptor in the component tests will cause several JS imports, which is caused by the metrics interceptor requiring the screens namespace which eventually imports all the screen imports for third-party modules.

So instead we configured shadow-cljs to and the interceptor namespace to exclude loading the metrics interceptor to avoid the extra imports that would require extra JS mocking for the component specs.